### PR TITLE
aur: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by gutjuri — APPROVED** [review]

File: pages.es/linux/aur.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# aur
+
+> Construye paquetes desde el AUR y gestiona repositorios locales.
+> Nota: Es necesario establecer un repositorio local en `/etc/pacman.conf` e instalar `vifm` para que esto funcione completamente.
```
Comment (tricantivu) [inline pages.es/linux/aur.md:4]: What does `esto` refer to?

File: pages.es/linux/aur.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> Más información: <https://github.com/aurutils/aurutils>.
+
+- Busca un paquete en la base de datos AUR:
```
Comment (tricantivu) [inline pages.es/linux/aur.md:7]:
```suggestion
- Busca un paquete en la base de datos del AUR:
```

File: pages.es/linux/aur.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`aur search {{palabra_clave}}`
+
+- Descarga un paquete y sus dependencias desde AUR, los compila y los añade a un repositorio local:
```
Comment (tricantivu) [inline pages.es/linux/aur.md:11]:
```suggestion
- Descarga un paquete y sus dependencias desde el AUR, los compila y añade a un repositorio local:
```

File: pages.es/linux/aur.md (hunk 1 above)
Comment (kant) [reply pages.es/linux/aur.md:4]: take it off `esto`

**Review by tricantivu — APPROVED** [review]

**Pull request merged**